src/renderer/fonts/font_options.rs

```diff
@@ -13,17 +21,47 @@ pub struct FontOptions {
     pub edging: FontEdging,
 }
 
+impl FontWithFeatures {
+    pub fn parse_family_desc(desc: impl AsRef<str>) -> FontWithFeatures {
+        let mut features = vec![];
+        let mut name = None;
+        // reverse string at first, then split_inclusive at '+' or '-', then
+        // reverse each item in the result
```

Comment:
I think this comment needs to include why you reverse it. If I understand it correctly it's to keep prefixes rather than suffixes.

Reply:
this comment updated